modules/api/src/main/java/org/apache/ignite/catalog/definitions/TableDefinition.java

```diff
@@ -78,6 +82,16 @@ private TableDefinition(
         this.keyClass = keyClass;
         this.valueClass = valueClass;
         this.indexes = indexes;
+
+        if (schemaName == null) {
```

Comment:
if schemaName can be null it need to be @Nullable ? but seems it can`t be null ? or i miss smth ?